sdks/wasm/library_mono.js

```diff
@@ -358,7 +358,33 @@ var MonoSupportLib = {
 	mono_wasm_fire_bp: function () {
 		console.log ("mono_wasm_fire_bp");
 		debugger;
-	}
+	},
+
+	// Win32S = 0,
+	// Win32Windows = 1,
+	// Win32NT = 2,
+	// WinCE = 3,
+	// Unix = 4,
+	// // Since NET 3.5 SP1 or silverlight
+	// Xbox = 5,
+	// MacOSX = 6,
+	mono_wasm_os_platform_id: function () {
+		if (ENVIRONMENT_IS_WEB) {
+			var platformenv = window.navigator.platform.toUpperCase();
+			if (platformenv.indexOf("MAC")>=0
```

Comment:
This kind of substring match is too fragile, I think you need to have whitespace at the beginning or end of the match patterns.

Reply:
Agree, I think we should return WebAssembly or something.

Reply:
We can setup another PlatformId for WebAssembly.